src/import-data/ImportDataDestination.ts

```diff
@@ -362,9 +359,9 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
               h(NewWorkspaceModal, {
                 requiredAuthDomain: requiredAuthorizationDomain,
                 cloudPlatform: getCloudPlatformRequiredForImport(importRequest),
-                renderNotice: ({ selectedBillingProject }) => {
+                renderNotice: () => {
                   const children: ReactNode[] = [];
-                  if (isProtectedData && selectedBillingProject?.cloudPlatform === 'AZURE') {
+                  if (isProtectedData) {
```

Comment:
This means that we will show the section in the new workspace modal for GCP workspaces. It hasn't been shown there before for GCP, so please include a screenshot of how it looks with the menagerie of other GCP information.

Reply:
yes, with cWDS coming online policies now will be enforced on GCP workspace as well. I will make sure to add a screenshot (it will be the same as what users see for Azure today)

Reply:
Can you also include a screenshot of what it looks once the GCP billing project is selected? That's when all the extra Google stuff is shown.

Reply:
@cahrens Since my code really only changes the import UI parts, could you be more specific? I included the access controls showing as yes (when the imported pfb expects it), is there something else you are looking for?

Reply:
added!